ogc/wfs-xml-query/README.md

```diff
@@ -2,18 +2,21 @@
 
 Load a WFS feature table using an XML query.
 
-![](WfsXmlQuery.png)
+![Image of load WFS with XML query](WfsXmlQuery.png)
 
 ## Use case
 
-ArcGIS Runtime `QueryParameters` objects can't represent all possible queries that can be made against a WFS feature service. For example, Runtime query parameters don't support wildcard searches. However, queries can be provided as raw XML strings, allowing access to query functionality not available with `QueryParameters`.
+Runtime `QueryParameters` objects can't represent all possible queries that can be made against a WFS feature service. For example, Runtime query parameters don't support wildcard searches. You can provide queries as raw XML strings, allowing you to access query functionality not available with `QueryParameters`.
```

Comment:
We should specify ArcGIS Runtime when first referring to runtime: it's probably obvious but since it's an Esri product, using the full terminology on it's first use is best practice (in the same way ArcGIS Pro, ArcGIS Online are used etc). Probably best to use it the second time too
```suggestion
ArcGIS Runtime `QueryParameters` objects can't represent all possible queries that can be made against a WFS feature service. For example, ArcGIS Runtime query parameters don't support wildcard searches. You can provide queries as raw XML strings, allowing you to access query functionality not available with `QueryParameters`.
```